modules/apps/jaggery/mdm/src/mdm/modules/device.js

```diff
@@ -51,7 +52,7 @@ deviceModule = function () {
             log.error("User object was not found in the session");
             throw constants.ERRORS.USER_NOT_FOUND;
         }
```

Comment:
There is no error logs in finally blocks in this PR.
I think you have mistaken becouse some of the cde is shrink.Click the expand icon on the left side to get the full code.